ui/src/shared/components/PostDetail.tsx

```diff
@@ -127,18 +134,21 @@ const PostDetail: React.FC<PostDetailProps> = function (props) {
             </div>
             <Actions>
               <FavouriteButton active={data.favourite} />
-              <Menu>
-                <StyledMenuList>
-                  <StyledMenuItem>
-                    <EditIcon />
-                    Edit
-                  </StyledMenuItem>
-                  <StyledMenuItem>
-                    <DeleteIcon />
-                    <DangerText>Delete Post</DangerText>
-                  </StyledMenuItem>
-                </StyledMenuList>
-              </Menu>
+              'Context:' { context?.profile?.id }  'User:' { user.id } | 'Data:' { category }
```

Comment:
![image](https://user-images.githubusercontent.com/3888250/94564856-10cd2b80-029b-11eb-813e-f34f82ff98a6.png)

Eh not sure what is going on here

Reply:
Oops meant for debugging purposes, have removed it